docs/src/content/en/reference/workflows/sendEvent.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Reference: Workflow.sendEvent() | Building Workflows | Mastra Docs"
```

Comment:
There wasn't a page for this. Is this correct?

Reply:
Yeah I don't think this is documented